translations/ui-bulk-edit/en.json

```diff
@@ -61,8 +61,8 @@
   "list.preview.table.status.false": "Inactive",
 
   "list.errors.title": "Errors",
-  "list.errors.info": "<b>{fileName}: {entries} entries * {matched} records matched * {errors} errors</b>",
-  "list.errors.infoProcessed": "<b>{fileName}: {entries} entries * {matched} records changed * {errors} errors</b>",
+  "list.errors.info": "<b>{fileName}: {entries} entries * {matched, number} records matched * {errors, number} errors</b>",
+  "list.errors.infoProcessed": "<b>{fileName}: {entries} entries * {matched, number} records changed * {errors, number} errors</b>",
```

Comment:
`{entries}` needs the `{entries, number}` treatment too. There are also a few others (lines 246, 347, 386, 389) containing numeric substitutions. Your call on whether to fix 'em all or just those that are specifically in the scope of this story.

Reply:
Hi @zburke, updated etries, and all poited lines.